docs/installation_guide.md

```diff
@@ -302,110 +348,118 @@ $ sudo docker-compose -f ./docker-compose.yml -f ./docker-compose.notary.yml -f
 Please check the [Docker Compose command-line reference](https://docs.docker.com/compose/reference/) for more on docker-compose.
 
 ### Persistent data and log files
+
 By default, registry data is persisted in the host's `/data/` directory.  This data remains unchanged even when Harbor's containers are removed and/or recreated.  
 
-In addition, Harbor uses *rsyslog* to collect the logs of each container. By default, these log files are stored in the directory `/var/log/harbor/` on the target host for troubleshooting.  
+In addition, Harbor uses [rsyslog](https://www.rsyslog.com/) to collect the logs from each container. By default, these log files are stored in the `/var/log/harbor/` directory on the target host for troubleshooting.  
 
-## Configuring Harbor listening on a customized port
-By default, Harbor listens on port 80(HTTP) and 443(HTTPS, if configured) for both admin portal and docker commands, you can configure it with a customized one.  
+## Configuring Harbor to listen on a customized port
 
-### For HTTP protocol
+By default, Harbor listens on ports 80 (HTTP) and 443 (HTTPS, if configured) for both admin portal and Docker commands. But you can also configure it to listen on custom ports. 
 
-1.Modify docker-compose.yml  
-Replace the first "80" to a customized port, e.g. 8888:80.  
+### HTTP
 
-```
-proxy:
+1. Modify `docker-compose.yml` and replace the first `80` with a custom port, e.g. `8888:80`.
+
+    ```yaml
+    proxy:
     image: goharbor/nginx-photon:v1.6.0
     container_name: nginx
     restart: always
     volumes:
-      - ./common/config/nginx:/etc/nginx:z
+    - ./common/config/nginx:/etc/nginx:z
     ports:
-      - 8888:80
-      - 443:443
+    - 8888:80
+    - 443:443
     depends_on:
-      - postgresql
-      - registry
-      - core
-      - portal
-      - log
+    - postgresql
+    - registry
+    - core
+    - portal
+    - log
     logging:
-      driver: "syslog"
-      options:  
+        driver: "syslog"
+        options:
         syslog-address: "tcp://127.0.0.1:1514"
         tag: "proxy"
-```
+    ```
 
-2.Modify harbor.cfg, add the port to the parameter "hostname"  
+2.Modify `harbor.cfg`, adding the port to the `hostname` parameter:
 
-```  
-hostname = 192.168.0.2:8888
-```
+    ```conf
+    hostname = 192.168.0.2:8888
+    ```
+
+3. Re-deploy Harbor using the instructions in the [Managing Harbor's Lifecycle](#managing-harbors-lifecycle) section.
 
-3.Re-deploy Harbor referring to previous section "Managing Harbor's lifecycle".
 ### For HTTPS protocol
```

Comment:
The previous section was renamed to `HTTP`, this one can probably be renamed to `HTTPS`